src/configure/test/suite/GitHubClient.test.ts

```diff
@@ -0,0 +1,135 @@
+import { GithubClient } from "../../clients/github/githubClient";
```

Comment:
this class is a very low level class and while it is good to have tests for these, it would be better to also have tests for the classes where these client functions are called?

Reply:
Actually I have written tests for both, functions of this class and as well as for the functions which are calling the functions of this class.